src/Data/Unrestricted/Internal/Instances.hs

```diff
@@ -0,0 +1,232 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LinearTypes #-}
+{-# LANGUAGE MagicHash #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Data.Unrestricted.Internal.Instances where
```

Comment:
Why do you prefer this to defining these instances in the files where the classes are defined?

Reply:
Oh, this should be documented in a comment: it avoids a circular dependence with `Ur` and `Movable`. Since many classic `Movable` instances depend on `Ur` and the definition of `Movable` depends on `Ur`. 

After separating the `Movable` instances, I thought it clean to separate the rest too.